docs/source/install.md

```diff
@@ -29,13 +29,28 @@ $ conda install -c conda-forge evalml
 ```
 ````
 
+````{tab} PyPI (M1 Mac)
+```console
+# See the EvalML with core dependencies only section
```

Comment:
could we link here?

Reply:
I don't think so since this is a code block